src/main/webapp/scripts/civic.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Extract the government officials from the civic API response.
+ */
+function extractOfficials(civicObj) {
+  let officials = new Object();
+  const levels = ["administrativeArea1", "administrativeArea2", "country", "international", "locality", "regional", "special", "subLocality1", "subLocality2"];
```

Comment:
[Nit] This is longer than 80 characters.  I would put each entry on its own line.

`
const level = [
  "adminArea1"
  "adminArea2"
  "country"
   ...
]`

Reply:
Huh, the code format didn't respect my newlines.  

const level = [ 
  "adminArea1,
  "adminArea2",
   "country",
  ... 
]

Reply:
And the regular format doesn't respect leading newlines...

Oh well, pretend each entry in the list is intended by 2 spaces.